### PR TITLE
FAT-C358990 (Spitfire)

### PR DESCRIPTION
**Pull request by ALegchilov**
[C358990](https://foliotest.testrail.io/index.php?/cases/view/358990)
[Allure](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/4388/allure/)

### CODE REVIEW

File: cypress/e2e/inventory/holdings/create-new-marc-holdings.cy.js [hunk 1]
```diff
@@ -156,3 +158,87 @@ describe('Create holding records with MARC source', () => {
...
+      Permissions.uiQuickMarcQuickMarcHoldingsEditorCreate.gui,
+      Permissions.uiQuickMarcQuickMarcHoldingsEditorAll.gui,
+    ]).then((createdUserProperties) => {
+      ServicePoints.createViaApi(servicePoint);
+      defaultLocation = Location.getDefaultLocation(servicePoint.id);
+      testData.tag852_B_value = `$b ${defaultLocation.code}`;
+      testData.tag852_B_E_values = `$b ${defaultLocation.code} $e Test`;
+      Location.createViaApi(defaultLocation);
+      user = createdUserProperties;
+      cy.loginAsAdmin({ path: TopMenu.dataImportPath, waiter: DataImport.waitLoading }).then(
```
Comment (dnovikau) [inline cypress/e2e/inventory/holdings/create-new-marc-holdings.cy.js:190]: Can this be done via API calls?

**Review by dnovikau — COMMENTED** [review]
I recommend to move this test to separate file. Test has it's own preconditions and doesn't match current file name.

File: cypress/e2e/inventory/holdings/create-new-marc-holdings.cy.js (hunk 1 above)
Comment (ALegchilov) [reply cypress/e2e/inventory/holdings/create-new-marc-holdings.cy.js:190]: I'll check

Comment (ALegchilov) [reply cypress/e2e/inventory/holdings/create-new-marc-holdings.cy.js:190]: Haven't find a way how to import marc file through API

**Review by zentestuken — COMMENTED** [review]
Also recommend to move this test to a separate file

**Review by zentestuken — APPROVED** [review]

**Pull request merged**